Don't fail, when the tester's home directory (or any sub-path
thereof) is a symbolic link...

	-mi

--- ../tests/fCmd.test	Sun Mar 19 17:47:30 2006
+++ ../tests/fCmd.test	Tue Jun 27 14:13:53 2006
@@ -578,5 +578,5 @@
 } {1 {error copying "td2" to "~/td1/td2": permission denied}}
 test fCmd-6.26 {CopyRenameOneFile: doesn't use original name} \
-	{unixOnly notRoot} {
+	-constraints {unixOnly notRoot} -match regexp -body {
     cleanup
     file mkdir ~/td1/td2
@@ -587,5 +587,5 @@
     file delete -force ~/td1
     set msg
-} "1 {error copying \"~/td1\" to \"td1\": \"[file join [file dirname ~] [file tail ~] td1 td2]\": permission denied}"
+} -result "1 {error copying \"~/td1\" to \"td1\": \".*[file join td1 td2]\": permission denied}"
 test fCmd-6.27 {CopyRenameOneFile: TclpCopyDirectory failed} \
 	{unixOnly notRoot xdev} {
